packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -204,7 +204,7 @@ export default {
   computed: {
     ...mapMobileObserver(),
     isCartEmpty() {
-      return !!this.cartItemsQty;
+      return !!parseInt(this.cartItemsQty);
```

Comment:
I think you should use radix, because it is recommended https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt :)

Reply:
thank you. Fixed :)